cradle-core/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
-	compile "org.slf4j:slf4j-api:${slf4j_version}"
-	compile "org.slf4j:slf4j-log4j12:${slf4j_version}"
+	implementation "org.slf4j:slf4j-api:${slf4j_version}"
+	testImplementation "org.slf4j:slf4j-log4j12:${slf4j_version}"
 	compile "com.fasterxml.jackson.core:jackson-databind:${jackson_version}"

```

Comment:
What about others dependecies? I think we should migrate all of them to `implementation` and `testImplementation`